src/components/UserProfile/ScheduleReasonModal/ScheduleReasonModal.jsx

```diff
@@ -65,6 +66,20 @@ const ScheduleReasonModal = ({
   },[numberOfReasons,infringementsNum])
 // ===============================================================
 
+  //use document.getElementById to get the element of role
```

Comment:
This is causing the issue. Role can be retrieved from the user object.

Reply:
Hi, I fixed this bug, please check it again. Thank you so much!